test/ButtonInputSpec.js

```diff
@@ -42,8 +42,6 @@ describe('ButtonInput', () =>{
     ReactTestUtils.renderIntoDocument(
       <ButtonInput value="button" bsStyle="danger" />
     );
-
-    console.warn.called.should.be.false;
```

Comment:
@taion let me know that the tests will fail on any unexpected warnings, so the test will still fail appropriately without the assertion.